src/store/createSarcophagus/actions.ts

```diff
@@ -0,0 +1,25 @@
+import { ActionMap } from '../ActionMap';
```

Comment:
This file will conflict with my pending PR's but we'll cross that bridge when we get there.